homeassistant/components/ferienapidotde/binary_sensor.py

```diff
@@ -0,0 +1,149 @@
+"""Binary sensor to indicate if today is a german vacation day or not.
+
+Utilizes the api `ferien-api.de` to provide a binary sensor to indicate if
+today is a german vacation day or not - based on your configured state.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.ferienapidotde/
+"""
```

Comment:
A one-line docstring is enough. It's no longer needed to add a link to the documentation.